app/views/conferences/groups/show.html.erb

```diff
@@ -0,0 +1 @@
+<%= react_component('GroupPage', { conferenceId: @conferenceId, id: @id }) %>
```

Comment:
What do you mean by this @warrenshen? We shouldn't need the conferenceId here at all right?